com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -695,26 +721,23 @@ else if (path.endsWith(".html")) {
             if (build_report_html != null) {
                 reportFiles.add(new File(build_report_html));
             }
-            TimeProfiler.init(reportFiles, false);
+            TimeProfiler.init(reportFiles, fromEditor);
```

Comment:
we need to remove second `TimeProfiler.init()` if `InvocationResult` is the only entry point we have for both the editor and the engine.

Reply:
Yeah, I thought about that. For now, we still have another entry point in the editor. When we change the editor to only use `Bob.invoke`, we could remove that `TimeProfiler.init()` call.